bootstrap/Makefile.bootstrap

```diff
@@ -0,0 +1,24 @@
+export SHELL = /bin/bash
+export TARDIGRADE_CI_ORG ?= plus3it
+export TARDIGRADE_CI_PROJECT ?= tardigrade-ci
+export TARDIGRADE_CI_BRANCH ?= master
+export TARDIGRADE_CI_PATH ?= $(shell until [ -d "$(TARDIGRADE_CI_PROJECT)" ] || [ "`pwd`" == '/' ]; do cd ..; done; pwd)/$(TARDIGRADE_CI_PROJECT)
+
+MAKEFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+PROJECT_NAME := $(notdir $(patsubst %/,%,$(dir $(MAKEFILE_PATH))))
+PROJECT_ROOT := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
```

Comment:
looks like this logic can reuse MAKEFILE_PATH...

```suggestion
PROJECT_ROOT := $(dir $(MAKEFILE_PATH))
```

Reply:
updated